client/my-sites/stats/wordads/highlights-section.jsx

```diff
@@ -0,0 +1,153 @@
+			heading: translate( 'Paid', {
+				comment: 'Total WordAds earnings that have been paid out',
+			} ),
+			amount: paid,
+		},
+		{
+			heading: translate( 'Outstanding amount', {
+				comment: 'Total WordAds earnings currently unpaid',
+			} ),
+			amount: owed,
+		},
+	];
+	// Transform/index the data for use with React.
+	return highlights.map( ( highlight, i ) => ( {
+		id: i,
+		formattedAmount: getAmountAsFormattedString( highlight.amount ),
+		icon: <Icon icon={ starEmpty } />,
+		...highlight,
+	} ) );
+}
+
+function payoutNotices( earnings ) {
+	const owed =
+		earnings && earnings.total_amount_owed ? numberFormat( earnings.total_amount_owed, 2 ) : '0.00';
+	const notice = {
+		id: 'notice',
+		value: translate(
+			'Outstanding amount of $%(amountOwed)s does not exceed the minimum $100 needed to make the payment.',
+			{
+				comment: 'WordAds: Insufficient balance for payout.',
+				args: { amountOwed: owed },
+			}
+		),
+	};
+	const limit = {
+		id: 'limit',
+		value: translate(
+			'Payment will be made as soon as the total outstanding amount has reached $100.',
+			{
+				comment: 'WordAds: Payout limit description.',
+			}
+		),
+	};
+	const payout = {
+		id: 'payout',
+		value: translate(
+			'Outstanding amount of $%(amountOwed)s will be paid approximately 45 days following the end of the month in which it was earned.',
+			{
+				comment: 'WordAds: Payout will proceed.',
+				args: { amountOwed: owed },
+			}
+		),
+	};
+	return owed < 100 ? [ notice, limit ] : [ payout ];
+}
+
+function HighlightsSectionHeader( props ) {
+	const [ isTooltipVisible, setTooltipVisible ] = useState( false );
+	const iconReference = useRef( null );
+	const localizedTitle = translate( 'Totals', {
+		comment: 'Heading for WordAds earnings highlights section',
+	} );
+	if ( ! props.notices || props.notices.length === 0 ) {
+		return <h1 className="highlight-cards-heading">{ localizedTitle }</h1>;
+	}
+	return (
+		<h1 className="highlight-cards-heading">
+			{ localizedTitle }{ ' ' }
+			<Gridicon
+				icon="info-outline"
+				ref={ iconReference }
+				onMouseEnter={ () => setTooltipVisible( true ) }
+				onMouseLeave={ () => setTooltipVisible( false ) }
+			/>
+			<Popover
+				className="tooltip tooltip--darker tooltip-wordads highlight-card-tooltip"
+				isVisible={ isTooltipVisible }
+				position="bottom right"
+				context={ iconReference.current }
+			>
+				<div className="highlight-card-tooltip-content">
+					{ props.notices.map( ( notice ) => (
+						<p key={ notice.id }>{ notice.value }</p>
+					) ) }
+				</div>
+			</Popover>
+		</h1>
+	);
+}
+
+function HighlightsListing( props ) {
+	return (
+		<div className="highlight-cards-list">
+			{ props.highlights.map( ( highlight ) => (
+				<HighlightCardSimple
+					key={ highlight.id }
+					heading={ highlight.heading }
+					icon={ highlight.icon }
+					value={ highlight.formattedAmount }
+				/>
+			) ) }
+		</div>
+	);
+}
+
+export default function HighlightsSection( props ) {
```

Comment:
Similar to the cards, would it be possible to put another variant of the header [here](https://github.com/Automattic/wp-calypso/tree/trunk/packages/components/src/highlight-cards)?

Reply:
I hadn't considered this. I was thinking of the Storybook components as components that might be reusable outside Stats. For example, I can see other teams needing to make use of simple highlight cards. The section components are a bit more data-coupled though, so it made sense they would live inside Stats.

Are we thinking we should develop entire sections or layouts inside the components package, even if we don't see them as reusable?

Reply:
I see your point now, and this file has many functions specific to ads. So let's keep it here.